src/migration/tasks/AddSiteAreaLimitToConsumptionsTask.ts

```diff
@@ -22,7 +22,7 @@ export default class AddSiteAreaLimitToConsumptionsTask extends MigrationTask {
   async migrateTenant(tenant: Tenant) {
     let modifiedCount = 0;
     // Get Charging Stations
-    const siteAreas = await SiteAreaStorage.getSiteAreas(tenant.id, { withChargingStations: true }, Constants.DB_PARAMS_MAX_LIMIT);
+    const siteAreas = await SiteAreaStorage.getSiteAreas(tenant.id, { withChargingStationsAttribute: true }, Constants.DB_PARAMS_MAX_LIMIT);
```

Comment:
What is the purpose of renaming this param to withChargingStationsAttribute?

Reply:
+ withChargingStationsAttribute add an attribute to the returned object containing the charging stations in the site area;
+ withChargingStations will only return site areas with charging stations attached to them. It follows the same semantic as getSites().

Reply:
> * withChargingStations

withChargingStations means: get the charging stations of the site areas and return it as a property (join).
If you want to get the Site Areas which have at least one charging station, maybe use something like this withOnlyChargingStations and provide the MongoDB request

Reply:
> > * withChargingStations
> 
> withChargingStations means: get the charging stations of the site areas and return it as a property (join).

That param that is used in getSites(), HttpSiteAreaRequest and now getSiteAreas() does not mean at all that in the code. I have just aligned to the current semantic in getSiteAreas() after the param rename you've done in the previous related PR on getSites(). 
Changing the current semantic is fine but that's not the purpose of that PR and there's really a lot more to do to align params semantic between the REST API and the MongoDB methods. It's another topic, a new issue. Just like the use of queries only in MongoDB methods.